### DISCUSSION TITLE
Hardware specs for 500 users

### DISCUSSION THREAD

**Discussion by german2209** [mynaparrot/plugNmeet-server#253] [General]
Hi,
Is it possible to handle a single meeting with 500 users, with only 1 or 2 webcams and screensharing during the meeting?
What should be the server specs to handle this amount of users?
Thanks in advanc for any guidance,
German

**Comment by jibon57** [comment]
Hi,
You'll require at least 16 cores server.

**Reply by german2209** [reply]
Thank you for your prompt response.
Wich one of the following options do you think should be the best option to handle max 500 users in a single room?

Option 1:
AMD Ryzen Pro 1700x (8 x 3,4 GHz) this handles 16 threads or 16 vCores
RAM: 32 GB DDR4 ECC
SSD: 2x120 GB

Option 2:
Processor: AMD Ryzen 7 3700x (8 x 3,9 GHz) this handles 16 threads or 16 vCores
RAM: 64 GB DDR4 ECC
SSD: 2x480 GB

Option 3:
2 x E5-2620v3 (6 x 2,4 GHz) this reports 24 threads or 24 vCores
RAM: 64 GB DDR4 ECC
SSD: 2x240 GB

Option 4:
Processor: AMD Epyc 7351P (16 x 2,9 GHz) this reports 32 threads or 32 vCores
RAM: 64 GB DDR4 ECC
SSD: 2x480 GB

Tanks in advance
